packages/dx-react-grid-material-ui/src/templates/table-header-cell/grouping-control.jsx

```diff
@@ -4,11 +4,15 @@ import classNames from 'classnames';
 import List from 'material-ui-icons/List';
 import { withStyles } from 'material-ui/styles';
 
-const styles = theme => ({
+export const styles = theme => ({
```

Comment:
Why we export `styles`?

Reply:
It allows using classes in UT.

Reply:
But you are using the `getClasses()` function for this purpose and do not import anything else but `GroupingControl` in [tests](https://github.com/DevExpress/devextreme-reactive/pull/692/files/4e6566767c538da4f40e26275582ed4e5e9a0c82#diff-2f5bd88e703ecd4193eabf4a066be058R3).

Reply:
Seems like, it doesn't require to export styles anymore.